CHANGES.md

```diff
@@ -22,6 +22,10 @@ This files lists the changes during the lifetime of this project.
 - (migration) Remove hours_per_week on service and placement
 - (migration) Make email unique in db
 - Fix that RVOMixin uses proper jinja environment (enabling components and other functions)
+- 114: Implement right-side panel with colleague and assignment details
+- 114: Add deep-linking support for panel URLs (/placements// and /placements assignment_id/)
+- 114: Add HTMX integration for dynamic loading panels
```

Comment:
the feature addition is already covered in the top changelog item. this gives the impression that this is a separate change on a different part of the program ( which i think is not the case? )
```suggestion
```